app/src/main/java/com/github/se/cyrcle/ui/profile/ViewProfileScreen.kt

```diff
@@ -183,49 +179,71 @@ fun ViewProfileScreen(
-                  }
+              // Display the TabLayout
+              TabLayout(
+                  userViewModel = userViewModel,
+                  parkingViewModel = parkingViewModel,
+                  reviewViewModel = reviewViewModel,
+                  navigationActions = navigationActions)
             }
           }
         }
       }
 }
 
+/** Tab layout that displays different sections based on the selected tab. */
 @Composable
-private fun FavoriteParkingsSection(
+private fun TabLayout(
     userViewModel: UserViewModel,
     parkingViewModel: ParkingViewModel,
+    reviewViewModel: ReviewViewModel,
     navigationActions: NavigationActions
 ) {
-  val favoriteParkings = userViewModel.favoriteParkings.collectAsState().value
+  var selectedTabIndex by rememberSaveable { mutableIntStateOf(0) }
+  val tabs =
+      listOf(
+          stringResource(R.string.view_profile_screen_favorite_parkings),
+          stringResource(R.string.view_profile_screen_my_reviews))
+
+  Column(
+      modifier = Modifier.fillMaxWidth().padding(top = 56.dp) // Adjust based on button height
+      ) {
+        TabRow(selectedTabIndex = selectedTabIndex, modifier = Modifier.testTag("TabRow")) {
+          tabs.forEachIndexed { index, title ->
+            Tab(
+                text = { Text(title) },
+                selected = selectedTabIndex == index,
+                onClick = { selectedTabIndex = index },
+                modifier = Modifier.testTag("Tab${title.replace(" ", "")}"))
+          }
+        }
 
-  Text(
-      text = stringResource(R.string.view_profile_screen_favorite_parking_title),
-      style = MaterialTheme.typography.titleLarge,
-      modifier = Modifier.testTag("FavoriteParkingsTitle"))
+        when (selectedTabIndex) {
+          0 -> FavoriteParkingsSection(userViewModel, parkingViewModel, navigationActions)
+          1 ->
+              UserReviewsSection(
+                  reviewViewModel, userViewModel, parkingViewModel, navigationActions)
+        }
+      }
+}
 
-  Spacer(modifier = Modifier.height(12.dp))
+@Composable
+private fun FavoriteParkingsSection(
+    userViewModel: UserViewModel,
+    parkingViewModel: ParkingViewModel,
+    navigationActions: NavigationActions
+) {
+  val favoriteParkings = userViewModel.favoriteParkings.collectAsState().value
 
   if (favoriteParkings.isEmpty()) {
     Text(
         text = stringResource(R.string.view_profile_screen_no_favorite_parking),
         style = MaterialTheme.typography.bodyMedium,
-        modifier = Modifier.testTag("NoFavoritesMessage"))
+        modifier = Modifier.testTag("NoFavoritesMessage").padding(16.dp))
   } else {
-    LazyRow(
-        modifier = Modifier.fillMaxWidth().testTag("FavoriteParkingList"),
-        horizontalArrangement = Arrangement.spacedBy(8.dp)) {
+    LazyColumn(
+        modifier = Modifier.fillMaxWidth().padding(16.dp).testTag("FavoriteParkingList"),
+        verticalArrangement = Arrangement.spacedBy(8.dp)) {
```

Comment:
I think this Vertical Arrangement here might be why the parkings all display in a vertical line? I understand it's for symmetry with the Reviews but honestly it just looks a bit off on the screen :/. Would it be possible to put 3 Parkings per line?

Reply:
I don't understand the remark. This is not intended to stay like this, described in the "Future Improvements" section of the PR description. The cards will be horizontally elongated, just like in the list screen.

Reply:
Did you see the picture I posted with my review? I just thought it looked really odd to have tiny things when scrolling, and I wasn't super convinced it should stay like this even with you mentioning in the PR description that the Parking Cards would be elongated. If you're sure this will be dealt with, though, I'll trust you.

Reply:
I mean, you would be suggesting I should rework their UI layout when a PR will be specifically dedicated to that. I purposedly didn't change them (trust me it urges me) to let that PR take care of it. They are the exact same, just in a Column Layout rather than Row to match the Tab layout

Reply:
Cool, knowing it's on the way is enough for me